custom_components/inpost_paczkomaty/coordinator.py

```diff
@@ -7,29 +7,43 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 
-from .api import CustomInpostApi
+from .api import InPostApiClient
 from .models import ParcelsSummary
 
 _LOGGER = logging.getLogger(__name__)
 
 
 class InpostDataCoordinator(DataUpdateCoordinator[ParcelsSummary]):
-    def __init__(
-        self, hass: HomeAssistant, mailbay_api_client: CustomInpostApi
-    ) -> None:
+    """Data coordinator for InPost parcels API."""
+
+    def __init__(self, hass: HomeAssistant, api_client: InPostApiClient) -> None:
+        """Initialize the data coordinator.
+
+        Args:
+            hass: Home Assistant instance.
+            api_client: InPost API client instance.
+        """
         super().__init__(
             hass,
             _LOGGER,
             name="InPost Paczkomaty data coordinator",
             update_interval=timedelta(seconds=30),
```

Comment:
Under the hood backend polls InPost API every 300s because of rate limiting - can we meet somewhere closer to 300 than 30?

Reply:
In my opinion no need. I executed a lot of requests and haven't actually been rate limited. Just now I've sent 100 requests within 10 seconds and wasn't rate limited. This integration makes a lot less requests. Therefore, in my opinion we can leave it as it is.

In future we can migrate this value to a config file, so users can set it by theirselves.